src/components/FlyoutMenu/FlyoutMenu.jsx

```diff
@@ -0,0 +1,117 @@
+import './_flyout-menu.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SettingsAdjust16 as SettingsAdjust } from '@carbon/icons-react';
+import { Tooltip, Button } from 'carbon-components-react';
```

Comment:
We can import these relative to the project instead of directly from `carbon-components-react` so existing and future updates/overrides we make are available to this component.

```suggestion
import Tooltip from '../Tooltip';
import Button from '../Button';
```